.gitattributes

```diff
@@ -1 +1 @@
-*.pbxproj binary
+*.pbxproj binary merge=union
```

Comment:
Can you leave a comment here explaining why we are adding this?

Reply:
Done in cf5a11a